### DISCUSSION TITLE
warning "The clean trie cache is not found.." while pruning

### DISCUSSION THREAD

**Discussion by daron4ever** [bnb-chain/bsc#1300] [Q&A]
Hi,

I have the following warning message while pruning.
```
WARNING!
The clean trie cache is not found. Please delete it by yourself after the
pruning. Remember don't start the Geth without deleting the clean trie cache
otherwise the entire database may be damaged!
```

Does anyone know how to delete it?

**Comment by mikhail-khalizev** [comment]
Try to remove folder `geth\triecache`.

**Reply by nabilmahat** [reply]
is it solved by deleting it?

**Comment by ChlimaDisruptive** [comment]
The whole folder?
Thank you!

I'll do that.

**Comment by booger44** [comment]
> Привет,
> 
> Во время обрезки появляется следующее предупреждающее сообщение.
> 
> ```
> WARNING!
> The clean trie cache is not found. Please delete it by yourself after the
> pruning. Remember don't start the Geth without deleting the clean trie cache
> otherwise the entire database may be damaged!
> ```
> 
> Кто-нибудь знает, как его удалить?

Привет ,у меня такая же проблемка, спасибо тебе что нашел ответ раньше меня !

**Comment by cod9ine** [comment]
Will contribute more later.
